mdtraj/core/topology.py

```diff
@@ -1169,6 +1177,8 @@ class Residue(object):
         The chain within which this residue belongs
     resSeq : int
         The residue sequence number
+    segment_id : str
+        An optional label for the segment to which this residue belongs
     """
 
     def __init__(self, name, index, chain, resSeq):
```

Comment:
It will also need to be added to the constructor here, probably with a default value of the empty string.